docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+services:
+
+  db:
+    image: postgres:10-alpine
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: app_db
+      POSTGRES_USER: app_user
+      POSTGRES_PASSWORD: postgres!secret
+  # Uncomment this if you want to persist the data.
+  # volumes:
+  #   - "./pgdata:/var/lib/postgresql/data"
+    volumes:
+      - ./create_table.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
Based on my reading, this only executes if the database has not yet been created ie. /data/postgres is empty. I think this is fine. Ideally, we could run migrations and what not from the application, but this should get the job done.

Reply:
Agreed, but that would require a bit more changes than the docker embedding and I feel it should be another issue and change